### PR TITLE
updated the read me to include setup instructions.

### PR DESCRIPTION
**Pull request by Mmalumbo**
Signed-off-by: mmalumbo <mmalumbo@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,51 @@
 # adx-log-worker
-An ADX worker used to log progress in other workers. 
+An ADX worker used to log progress in other workers.
+
+# Email Worker
```
Comment (bwighane) [inline README.md]: ADX validation worker

File: README.md [hunk 2]
```diff
@@ -1,2 +1,51 @@
...
+
+Email worker is a background process that runs against email queue. It sends emails based on the data read in the queue
```
Comment (bwighane) [inline README.md]: This describes the email worker

**Review by bwighane — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Mmalumbo) [reply README.md]: removed.

File: README.md (hunk 1 above)
Comment (Mmalumbo) [reply README.md]: updated.

**Review by bwighane — APPROVED** [review]

**Pull request merged**